### ISSUE TITLE
Log the validation loss when using CLI while training an omnipose model

### ISSUE THREAD

**Issue by lacan** [kevinjohncutler/omnipose#86]
Looking through the log and the documentation at https://omnipose.readthedocs.io/training.html

I do not see how I can access the validation loss when training a new omnipose model. It would be necessary to see the performance of my model by looking at how both the training loss and validation loss change over time per epoch.

Could this please be added to the verbose outpus of the log when training an omnipose model using the CLI?

Otherwise, how do I access it via Python?

This is somewhat linked to #16

**Comment by kevinjohncutler** [comment]
I agree that the validation loss, if computed, should be added to the logs.
